compiler/circle-opselector/README.md

```diff
@@ -0,0 +1,58 @@
+# Circle Op-Selector
+
+`Circle Op-Selector` is a tool for creating new circle model by various methods to select nodes.
+
+## Example
+
+### 1. Select from location numbers
+
+```bash
+./opselector --by_id "1-3,5" --input input.circle --output output.circle
+```
+
+Then, output.circle which has node 1, 2, 3 and 5 will created.
+
+### 2. Select from node names
+
+```bash
+./opselector --by_name "Add_1,Sub_1,Concat_2" --input input.circle --output output.circle
+```
+
+Then, output.circle which has node Add_1, Sub_1 and Concat_2 will created.
+
+## Options

```

Comment:
`Opionts` and `Directory Structure` look good to me. But, I think we might as well not have them in this `README.md` because they are kind of burden to us.

If they are changed, developers should change this `README.md` as well, which is easy to forget and a bit bothers developers.

Reply:
Adding more example would be helpful instead. Above examples don't have `select`, `deselect` things yet.

Reply:
+1 for removing Directory Structure. But this document should explain how to use (including option). I usually copy and paste the `--help` + some examples.

Reply:
We think it might be better that there are not options `select`, `deselect` now.
So from next time on, we would temporarily remove these options.